src/utils/components.ts

```diff
@@ -9,8 +9,8 @@ import { toCamelCase } from '../utils/snakeCase.ts'
 export function transformComponent(
   d: MessageComponentData[]
 ): MessageComponentPayload[] {
-  return d.map((e: any) => {
-    e = { ...e }
+  return d.map((_: unknown) => {
+    const e = { ...(_ as Record<string, unknown>) }
```

Comment:
don't use `_` for the variables that is used

Reply:
done

Reply:
naisu